add empirical and jackknife covariance (#215)
openpharma/mmrm: NEWS.md

### New Features

- Add Kenward-Roger support for spatial covariance structures.

> - Add empirical and empirical Jackknife covariance estimator for $\beta$.

Reviewer: ```suggestion
- Add empirical and empirical Jackknife adjusted coefficients covariance matrix.
```
(to parallel the wording below)

>   In addition, the argument `method` now only stands for the method used
>   for degree of freedom, another argument `cov` is added to control the
>   method used to calculate the covariance for $\beta$.

Reviewer: ```suggestion
  In addition, the argument `method` now only specifies the method used
  for the degrees of freedom, another argument `cov` is added to specify the
  method used to adjust the coefficients covariance matrix.
```

> - Add optional tidymodels support.

Reviewer: ```suggestion
- Add optional `tidymodels` framework support.
```
